### PR TITLE
bpf/cpu: format code

### PR DESCRIPTION
**Pull request by Sylfrena**
pkg/pprof: fix log level

### CODE REVIEW

**Review by kakkoyun — APPROVED** [review]

**Review by brancz — APPROVED** [review]

File: pkg/pprof/pprof.go [hunk 1]
```diff
@@ -352,14 +351,8 @@ func (c *Converter) addExecutableInfo(
...
 	ei, err := processMapping.ExecutableInfo(addr)
-	if err != nil {
-		if !(os.IsNotExist(err) || errors.Is(err, fs.ErrNotExist)) {
-			// We don't want to log the error if the file doesn't exist,
-			// because it's probably a very short-lived process that weren't fast enough to obtain the FDs
-			level.Debug(c.logger).Log("msg", "failed to get executable info", "address", fmt.Sprintf("%x", addr), "err", err)
-		} else {
-			level.Warn(c.logger).Log("msg", "failed to get executable info", "address", fmt.Sprintf("%x", addr), "err", err)
-		}
+	if err != nil && errors.Is(err, fs.ErrNotExist) {
```
Comment (kakkoyun) [inline pkg/pprof/pprof.go:354]: Maybe these changes should at least be in another commit. WDYT?
